### ISSUE TITLE
Issue ID: DAC_Missing_Language_Attribute1 - Feature: About

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#4497]
Report link: https://drive.google.com/file/d/1kqw9ef8LtshwHVF_1ccS1mFkUCmGDh8A/view

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by oliverlloyd** [comment]
Closing this as migrating this page to DCR will fix this problem
